style.css

```diff
@@ -577,6 +561,70 @@ form {
   border-color: #2230d2;
 }
 
+/* Mobile Menu */
+#mobile-menu {
+  display: none; /* Initially hidden */
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(50, 70, 255, 0.8);
+  backdrop-filter: blur(10px); /* Apply a blur effect to the backdrop */
+  z-index: 9999; /* On top of other content */
+  padding: 86px 28px; /* Updated padding */
+  box-sizing: border-box;
+  overflow: auto;
+}
```

Comment:
- [OPTIONAL] Comments are best used selectively and strategically so that they do not make readability difficult, and also to avoid bugs. If you have gotten some assistance from AI technologies, I think removing the comments meant as an explanation would be a rewarding exercise for you to keep your code base clean.